spec/factories/event_factory.rb

```diff
@@ -1,12 +1,8 @@
 FactoryGirl.define do
   factory :event do
-    sequence :name do |n|
-      "event##{n}'s name"
+    sequence :event_id do |n|
+      "event##{n}"
     end
-    description '123'
-    user 'user'
-    start_time Time.new(2016, 0o1, 0o1)
-    end_time Time.new(2016, 0o1, 0o1) + 10.minutes
     association :conference_room, factory: :conference_room
```

Comment:
I think you can just write `conference_room` and that's it. Definitely don't need to specify factory if it has the same name